package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "private": true,
+  "devDependencies": {
+    "babel-core": "^6.26",
+    "babel-jest": "^23.4",
+    "babel-preset-env": "^1.7",
+    "eslint": "^5.5",
+    "eslint-config-airbnb-base": "^13.1",
+    "eslint-plugin-import": "^2.14",
+    "eslint-plugin-jest": "^21.22.0",
+    "expect": "^23.5",
+    "jest": "^23.5",
+    "jest-environment-node": "^23.4",
+    "p-retry": "^2.0",
+    "promise-fs": "^1.3",
+    "regenerator-runtime": "^0.12.1",
+    "request": "^2.88",
+    "request-promise-native": "^1.0",
+    "uuid": "^3.3.2"
+  },
+  "scripts": {
+    "pretest": "eslint --ignore-path .gitignore .",
+    "test": "jest"
```

Comment:
`jest --verbose` will get your tests listed, opposed to just the test-cases, will also give you millisecond timings on tests so you can see at a glance if any cases take longer

Reply:
Only just switched to multiple files so hadn't spotted it'd changed!